boards/shields/mikroe_silent_step_2_click/mikroe_silent_step_2_click.overlay

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2025 Navimatix GmbH
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&mikrobus_i2c {
+	status = "okay";
+	tca9538a_mikroe_silent_step_2_click: tca9538a@70 {
+		status = "okay";
+		compatible = "ti,tca9538";
```

Comment:
This does not even match with whats present on the board. The board has a `PCA9538A`.  How did this even work?